tardis/io/tests/test_model_reader.py

```diff
@@ -0,0 +1,35 @@
+import tardis
+from tardis.io.model_reader import (read_artis_density,
+read_simple_ascii_abundances)
+
```

Comment:
both of you are right the way that this is broken is wrong, but it needs to be broken onto two lines. Pycharm should do it right.